docker.go

```diff
@@ -770,9 +773,26 @@ func NewDockerProvider(provOpts ...DockerProviderOption) (*DockerProvider, error
 		config:                tcConfig,
 	}
 
+	logDockerServerInfo(p)
+
 	return p, nil
 }
 
+func logDockerServerInfo(p *DockerProvider) {
+	type empty struct{}
+	info := `%v - Connected to docker: 
+  Server Version: %v
+  API Version: %v
+  Operating System: %v
+  Total Memory: %v MB
+`
+
+	sv, _ := p.client.ServerVersion(context.Background())
+	v, _ := mem.VirtualMemory()
```

Comment:
Won't this give you the memory of the process executing the tests, rather than the memory of the Docker host?